packages/ui-svelte/eslint.config.js

```diff
@@ -1,3 +1,3 @@
-import config from '@toolchain/eslint-config/profile/node'
+import config from '@toolchain/eslint-config/profile/svelte'
```

Comment:
This was the issue. No `svelte` files were being listed because of this in this package.